internal/integration/bounce_rule_test.go

```diff
@@ -0,0 +1,211 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/go-sql-driver/mysql"
+
+	"github.com/jimmyjames85/bouncecm/internal/models"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type BounceRuleSuite struct {
+	suite.Suite
+
+	handler http.HandlerFunc
+	client  *http.Client
+}
+
+func (suite *BounceRuleSuite) SetupSuite() {
+	suite.client = &http.Client{}
+}
+
+func (suite *BounceRuleSuite) SetupTest() {
+	_, err := Database.Exec(`
+		CREATE TABLE IF NOT EXISTS bounce_rule (
+			id smallint(5) unsigned NOT NULL AUTO_INCREMENT,
+			response_code smallint(5) unsigned NOT NULL DEFAULT '0',
+			enhanced_code varchar(16) NOT NULL DEFAULT '',
+			regex varchar(255) NOT NULL DEFAULT '',
+			priority tinyint(3) unsigned NOT NULL DEFAULT '0',
+			description varchar(255) DEFAULT NULL,
+			bounce_action varchar(255) NOT NULL,
+			PRIMARY KEY (id),
+			UNIQUE KEY bounce_rule_components (response_code,enhanced_code,regex)
+		) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+	`)
+
+	if err != nil {
+		suite.T().Fatalf("Failed to setup for test\nError: %v", err)
```

Comment:
Funtionally speaking this works fine. But since you're already using the `testify/assert` package then you can try looking into `assert.NoError` for things like this. I'd go one step further and checkout the `testify/require` package which is identical to `assert`, except it terminates the test immediately if it fails (useful for things that must succeed).

Looking at some docs, it looks like `suite` also exposes all the assert functions. Should be able to do something like `suite.NoError` and `suite.Equal` 

Same comment for all the other `err != nil` checks. Not necessary, but it makes things a little nicer